template/src/api/auth.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthTokens } from './common'
+import { Credentials } from './types/auth.types'
+
+// This is the place where you want to call your auth provider API
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
You can remove this eslint and rename:

 - `credentials` to `_credentials`,
 - `refreshToken` to `_refreshToken`.
 
Now if somebody uses this template and wants to modify `logIn` and `refreshTokens` they can miss `eslint-disable` line. I think it will be better with underscore prefixes.